src/controller/CHIPCommissionableNodeController.h

```diff
@@ -27,23 +28,19 @@ namespace chip {
 
 namespace Controller {
 
-constexpr uint16_t kMdnsPort = 5353;
-
-class DLL_EXPORT CommissionableNodeController : public Mdns::ResolverDelegate
+class DLL_EXPORT CommissionableNodeController : public AbstractMdnsDiscoveryController
 {
 public:
     CommissionableNodeController(){};
     virtual ~CommissionableNodeController() {}
 
     CHIP_ERROR Init();
 
-    CHIP_ERROR DiscoverAllCommissionersLongDiscriminator(uint16_t long_discriminator);
+    CHIP_ERROR DiscoverAllCommissioners(uint16_t long_discriminator);
```

Comment:
I know the name sounds weird, and it's possible to differentiate the function signatures in this case. However, the intent of the explicit naming was so we could discover the rest of the subtypes in the future. ex. short discriminator, vendor, etc. The function signatures will be the same as the long (uint16 value as the input).

Another possibility is to expose the DiscoveryFilter explicitly so we can use the enum value.

Reply:
Got it. Went with adding the names of the subtypes in the function names.